dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
```

Comment:
You also need <WindowsSdkPackageVersion>10.0.18362.22-preview</WindowsSdkPackageVersion>

Reply:
@manodasanW is the new runtime already in the new SDK 5.0.402? I'd assume no as name has preview in it. If not, and we do a direct sdk ref, do we still need to update SDK to 5.0.402?

Reply:
The .NET SDK doesn't have this package yet which as you noted by the preview.  The .NET SDK update is here because there was .NET runtime fixes for CsWinRT related scenarios which you probably might want to have for any tests or so.